packages/jellyfish-api-core/__tests__/category/blockchain/getChainTxStats.test.ts

```diff
@@ -0,0 +1,90 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+
+describe('ChainTxStats', () => {
+  const testing = Testing.create(new MasterNodeRegTestContainer())
+
+  beforeAll(async () => {
+    await testing.container.start()
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  it('should getChainTxStats for selected interval', async () => {
+    const blockHash = await testing.misc.waitForBlockHash(3)
+    const stats = await testing.rpc.blockchain.getChainTxStats(2, blockHash)
+
+    expect(stats.time).toBeGreaterThan(1550000000)
+    expect(stats.txcount).toStrictEqual(12)
+    expect(stats.window_final_block_hash.length).toStrictEqual(64)
+    expect(stats.window_final_block_height).toStrictEqual(3)
+    expect(stats.window_block_count).toStrictEqual(2)
+    expect(stats.window_tx_count).toStrictEqual(2)
+    expect(stats.window_interval).toStrictEqual(2)
+    expect(stats.txrate).toStrictEqual(1)
+
+    expect(typeof stats.time).toStrictEqual('number')
+    expect(typeof stats.txcount).toStrictEqual('number')
+    expect(typeof stats.window_final_block_hash).toStrictEqual('string')
+    expect(typeof stats.window_final_block_height).toStrictEqual('number')
+    expect(typeof stats.window_block_count).toStrictEqual('number')
+    expect(typeof stats.window_tx_count).toStrictEqual('number')
+    expect(typeof stats.window_interval).toStrictEqual('number')
+    expect(typeof stats.txrate).toStrictEqual('number')
```

Comment:
i believe since toStrictEqual was already used above, there is no need to check for the type again

Reply:
You're right! Mind stuck in JS 🤪